tests/test_scripts.py

```diff
@@ -286,6 +331,33 @@ def test_train_adversarial(tmpdir, named_configs):
     _check_train_ex_result(run.result)
 
 
+def test_train_adversarial_sac(tmpdir):
```

Comment:
It's a bit sad how similar this is to `test_train_adversarial`, but there are enough differences (pendulum vs cartpole, additional named configs, additional assertion) I think you made the right call just copy-pasting and editing.

Reply:
Simplified something that has been tested above.